tests/tests_fabric/test_connector.py

```diff
@@ -1088,95 +1086,3 @@ def test_xla_fsdp_automatic_strategy_selection(monkeypatch, tpu_available):
 
     if added_fsdp:
         strategies.STRATEGY_REGISTRY.pop("fsdp")
-
-
-def test_connector_transformer_engine(monkeypatch):
```

Comment:
This test is moved to its own file

Reply:
My suggestion is to have the connector assertions here, while doing plugin specific unit tests in the respective file